doc/plotting.rst

```diff
@@ -23,6 +23,11 @@ Matplotlib syntax and function names were copied as much as possible, which
 makes for an easy transition between the two.
 Matplotlib must be installed before xarray can plot.
 
+To use xarray's plotting capabilities with time coordinates containing
+``cftime.datetime`` objects
+`nc-time-axis <https://github.com/SciTools/nc-time-axis>` v1.2.0 or later
```

Comment:
I think [these kinds of links require a trailing underscore in reStructuredText](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#embedded-uris-and-aliases).
```suggestion
`nc-time-axis <https://github.com/SciTools/nc-time-axis>`_ v1.2.0 or later
```